From b71556896e2f424b9930fd24c2ff1e749223f70e Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 27 Oct 2020 11:01:32 +0100
Subject: [PATCH] luci-app-transmission: add dependency for transmission-daemon

In recent changes in packages feed
(https://github.com/openwrt/packages/commit/56e4edad82211c14528122566f56168f1bee0fd8),
there were removed variants, so if you will install
luci-app-transmission now, it will pull also transmission-daemon
and users do not need to install it separately.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 applications/luci-app-transmission/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/applications/luci-app-transmission/Makefile b/applications/luci-app-transmission/Makefile
index 1a1166f2d..0696bdac8 100644
--- a/applications/luci-app-transmission/Makefile
+++ b/applications/luci-app-transmission/Makefile
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=LuCI Support for Transmission
-LUCI_DEPENDS:=+luci-compat +luci-lib-ipkg
+LUCI_DEPENDS:=+luci-compat +luci-lib-ipkg +transmission-daemon
 
 include ../../luci.mk
 
-- 
2.17.1

